tests/actions/draftActions.test.js

```diff
@@ -51,4 +59,54 @@ describe('Draft Actions', () => {
       expect(store.getActions()).toEqual(expectedActions);
       done();
     });
+
+  it('should remove a test from the draft', async (done) => {
+    const order = {
+      display: 'hermatocrite',
+      uuid: '2343564321'
+    };
+
+    const draftType = 'single';
+    const expectedActions = {
+      DELETE_TEST_FROM_DRAFT_LAB_ORDER,
+      order
+    }
+
+    const store = mockStore({});
+    await store.dispatch(discardTestsInDraft({ order, draftType }), () => {
+      expect(store.getActions()).toEqual(expectedActions);
+    });
+    done();
+  });
+
+  it('should remove a test panel from the draft list', async (done) => {
+    const order = {
+      display: 'hermatocrite',
+      uuid: '2343564321'
+    };
+
+    const draftType = 'panel';
+    const expectedActions = {
+      DELETE_PANEL_FROM_DRAFT_LAB_ORDER,
+      order
+    }
+
+    const store = mockStore({});
+    await store.dispatch(discardTestsInDraft({ order, draftType }), () => {
+      expect(store.getActions()).toEqual(expectedActions);
+    });
```

Comment:
The feedback https://github.com/openmrs/openmrs-owa-orderentry/pull/172#discussion_r214695126 also applies here.

Reply:
Thanks, corrected.